src/software/new_geom/BUILD

```diff
@@ -0,0 +1,29 @@
+package(default_visibility = ["//visibility:public"])
+
+cc_library(
+    name = "new_geom",
+    srcs = [
+        "polynomial.cpp",
+    ],
+    hdrs = [
+        "polynomial.h",
+    ],
+    copts = [
+        "-std=c++17",
+    ],
```

Comment:
We shouldn't have to add this, everything in the codebase should already be built with it. Does removing this cause an error?

Reply:
Removing it caused no errors, removed in: 8c415dfe22b7c11d92acd4f9cfd53990d5ac821f